docker/shadow-hand/kinetic/base/Dockerfile

```diff
@@ -1,3 +1,9 @@
+#
+# Kinetic with build tools Dockerfile
```

Comment:
Does this sound right?
LABEL Description="This Kinetic image contains ROS and Shadow's hand software with build tools. It includes IDE environments" Vendor="Shadow Robot" Version="1.0"